File: C++/BroCode/WhileLoop.cpp
#include <iostream>
#include <string>

int main() 
{
   std::string name;
   
   while(name.empty()){
      std::cout << "Enter your name: ";
      std::getline(std::cin, name);
   }
   
   std::cout << "Hello " << name;

   return 0;
}

File: C#/BroCode/WhileLoop.cs
using System;

namespace MyFirstProgram
{
    class Program
    {
        static void Main(string[] args)
        {

            // while loop = repeats some code while some condition remains true

            String name = "";

            while (name == "")
            {
                Console.Write("Enter your name: ");
                name = Console.ReadLine();
            }

            Console.WriteLine("Hello " + name);

            Console.ReadKey();
        }
    }
}
